README.md

```diff
@@ -2,20 +2,31 @@
  
 # ⚒️ Metaflow Admin Tools
 
-This repository contains various configuration files, tools and utilities for operating [Metaflow](https://github.com/Netflix/metaflow) in production. See [Metaflow documentation](https://docs.metaflow.org) for more information about Metaflow architecture. The tools included in this repo include:
-
-## Metaflow Cloudformation template
+This repository contains various configuration files, tools and utilities for operating [Metaflow](https://github.com/Netflix/metaflow) in production. See [Metaflow documentation](https://docs.metaflow.org) for more information about Metaflow architecture. Top level folders are structured as follows:
 
+## Metaflow on AWS (/aws)
+### Metaflow Cloudformation template
 If you're not already using Terraform, this is the easiest way to get started on AWS. You can find the template under [aws/cloudformation](./aws/cloudformation) in this repository.
 
-## Terraform modules for AWS
+### Sample Terraform templates for AWS
+An alternate infrastructure as code option is Terraform. An end-to-end example can be found under [aws/terraform](./aws/terraform) in this repository. The example leverages the official Terraform module [outerbounds/terraform-aws-metaflow](https://registry.terraform.io/modules/outerbounds/metaflow/aws/latest) as a building block.
```

Comment:
Purely phrasing nit: I found the first sentence kinda hard to parse, had to re-read it a couple times ("an alternate infrastructure.. what ?"). Maybe its just me, but something like this would be easier for me:

```suggestion
Another deployment option is using Terraform. An end-to-end example can be found under [aws/terraform](./aws/terraform) in this repository. The example leverages the official Terraform module [outerbounds/terraform-aws-metaflow](https://registry.terraform.io/modules/outerbounds/metaflow/aws/latest) as a building block.
```